### PR TITLE
[CLEANUP] Remove {{render}}

### PR DESCRIPTION
**Pull request by thoov**
Related: [ember-2-legacy PR](https://github.com/emberjs/ember-2-legacy/pull/21)

Apart of: #15876

### CODE REVIEW

**Comment by thoov** [comment]
@rwjblue I don't have a great way of getting rid of these other files other than putting a comment on them for future removal. Do you have any ideas?

Right now I'm just making sure that we don't include render is what we want just not sure if we can clean this up any more

File: packages/ember-glimmer/lib/syntax.ts [hunk 1]
```diff
@@ -65,7 +65,11 @@ export function registerMacros(macro: any) {
 export function populateMacros(blocks: any, inlines: any) {
   inlines.add('outlet', outletMacro);
   inlines.add('component', inlineComponentMacro);
-  inlines.add('render', renderMacro);
+
+  if (ENV._ENABLE_RENDER_SUPPORT === true) {
```
Comment (rwjblue) [inline packages/ember-glimmer/lib/syntax.ts]: I think this `ENV` is not imported (thats what one of the test failures indicated at least)..

**Review by rwjblue — COMMENTED** [review]
The approach seems good to me, I don't think there is much more we can do to reduce payload size while still supporting what we need...

File: packages/ember-glimmer/lib/syntax.ts (hunk 1 above)
Comment (rwjblue) [reply packages/ember-glimmer/lib/syntax.ts]: @thoov - I think fixing this will fix CI.

Comment (thoov) [reply packages/ember-glimmer/lib/syntax.ts]: @rwjblue Yep that fixed it. Should be good to go now

**Review by rwjblue — APPROVED** [review]

**Pull request merged**